spec/services/solidus_square/customers/create_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe ::SolidusSquare::Customers::Create do
+  subject(:service) do
+    described_class.call({ client: '', spree_user: spree_user, spree_address: spree_address })
+  end
+
+  let(:spree_user) { create(:user_with_addresses) }
+  let(:spree_address) { spree_user.addresses.first }
+
+  context 'when customer already exists' do
+    before do
+      allow_any_instance_of(described_class).to receive(:search_customer).and_return({ 'id' => 111 })
```

Comment:
Do you mind replacing these `allow_any_instance_of``s in favor of Double and specific allow?